sphinx/util/pycompat.py

```diff
@@ -103,3 +103,40 @@ def execfile_(filepath, _globals):
         else:
             raise
     exec_(code, _globals)
+
+# ------------------------------------------------------------------------------
+# Internal module backwards-compatibility
+
+import warnings
+
+from six import class_types
```

Comment:
I am not sure — do you have any change proposals?  Otherwise I'll just merge it into master tomorrow (+ a changelog entry.)